docs/usage.md

```diff
@@ -0,0 +1,187 @@
+
+```ts
+space.on('membersUpdate', (members) => {
+  console.log(members);
+});
+```
+
+The following is an example `membersUpdate` event received by subscribers when a user enters a space:
+
+```json
+[
+  {
+    "clientId": "clemons@slides.com",
+    "isConnected": true,
+    "lastEvent": {
+      "name": "enter",
+      "timestamp": 1677595689759
+    },
+    "location": null,
+    "profileData": {
+      "username": "Claire Lemons",
+      "avatar": "https://slides-internal.com/users/clemons.png"
+    }
+  }
+]
+```
+
+The following are the properties of a `spaceMember` event:
+
+| Property | Description | Type |
+| -------- | ----------- | ---- |
+| clientId | The client identifier for the user. | string |
+| isConnected | Whether the user is connected to Ably or not. | boolean |
+| lastEvent | The most recent event emitted by the user and its timestamp. Events will be either `enter` or `leave`. | {name: string, timestamp: number} |
+| location | The current location of the user within the space. | any |
+| profileData | Optional user data that can be attached to a user, such as a username or image to display in an avatar stack. | object |
+
+To stop subscribing to member events, users can call the `space.off()` method.
+
+### Request member updates
+
+In addition to subscribing to events to see who joins and leaves a space, it is also possible to query the membership of a space in a one-off request using `getMembers()`. This returns an array of `spaceMember` objects. 
+
+The `getSelf()` method can be used to return the `spaceMember` object for the local client.
+
+## Enter a space
+
+When a user enters a space they should call `enter()`. This publishes an event to all subscribers of that space.
+
+`space.enter()` can take an optional object called `profileData` so that users can include convenient metadata to update an avatar stack, such as a username and profile picture.
+
+The following is an example of entering a space with `profileData`:
+
+```ts
+space.enter({
+  username: "Claire Lemons",
+  avatar: "https://slides-internal.com/users/clemons.png",
+});
+```
+
+### Leave a space
+
+A leave event is sent when a user leaves a space. This can occur for one of the following reasons:
+
+* `space.leave()` is called explicitly.
+* The user closes the tab.
+* The user is abruptly disconnected from the internet for longer than 2 minutes.
+  * Note that the time before they are seen has having left the space is configurable using `offlineTimeout`.
+
+## Subscribe to location updates
+
+Subscribe to `locationUpdate` events in order to display where users are within a space, such as which slide number they are currently viewing, or which cell or component they have selected. `locationUpdate` events for a space are sent when a user calls `locations.set()`, which can be configured to fire for whichever UI interactions you'd like. Use the `locations.on()` method to register a listener for `locationUpdates` events.
```

Comment:
Worth clarifying (as there's some confusion here admittedly) that locations.set is called to set your own location in a space so others can know and update their UI accordingly.